mkdocs.yml

```diff
@@ -51,9 +51,9 @@ nav:
     - Best Practices:
       - Executing Commands: guidance/pod-command.md
       - Worker Reconnection: best-practice/worker-head-reconnection.md
-  - Troubleshooting: troubleshooting.md
-  - Design:
-    - Core API and Backend Service: design/protobuf-grpc-service.md
+  - Troubleshooting:
+    - FAQ: guidance/FAQ.md
```

Comment:
Nit: Would it be better to make the FAQ top-level instead of under Troubleshooting? Currently it seems all the questions are troubleshooting-related, but my guess is we'll probably add some non-troubleshooting questions to the FAQ in the future.